meshPointExample.py

```diff
@@ -26,17 +28,21 @@
     # get the project object, list of mesh names, list of counting points
     project = sy.get_project(PROJECT_PATH)
     mesh_names = project.impl.GetMeshNamesAndSizes(EXPERIMENT_NAME)
+    #get_counting_points exports the x,yz, coordinates in voxel units from the bottom left corner

```

Comment:
```suggestion
    # TODO: get_counting_points exports the x,yz, coordinates in voxel units from the bottom left corner
```
Just adding a TODO here for clarity